### ISSUE TITLE
Error for archiving in Linux - s.join is not a function

### ISSUE THREAD

**Issue by quentinDupont** [ransome1/sleek#623]
## Bug Report

**App Version:** 2.0.3

**Platform:** Linux

**Installation Method:** Snap

**Bug Description:**
Clicking on Archive doesn't work and send an error "s.join is not a function" 
![image](https://github.com/ransome1/sleek/assets/9005817/ba9c173d-2ee6-48c7-897b-036ac90c8895)

Thanks for all the work :clap:

**Comment by ransome1** [comment]
@quentinDupont thanks for reporting this. 

1. Did you do a fresh install of sleek using Snap or had it been installed as an update?
2. Would you be willing to share your config file or at least the `files` section of it? If you press alt key, the window menu should appear, here you can click `sleek` and then `About`. A window appears and shows a **button** to reveal the location of the configuration in your file manager. From there navigate to `sleek/userData/config.json`.
3. Can I ask you to quit the application and  remove the userData folder and then restart sleek again, to see if this fixes the issue? This of course will delete all your settings.
4. If this doesn't help, can you please download the AppImage of the same release and tell me, if the issue also occurs here? https://github.com/ransome1/sleek/releases/tag/v2.0.3

**Comment by p-beattie** [comment]
Sleek 2.0.3 (flatpak update) on Linux Mint 22.1.  I experienced the same error.
Following your instructions, I quit the application and removed the userData folder, then restarted sleek again.
Identified my todo.txt file and my archive file (todo-done.txt) as requested at startup of sleek.
Created a test item, closed it.  Tested the archive function.  The issue no longer occurs for me.  
Checked Archive file contents - It was updated OK with the test item.
Thank you.
